IMAC-projects/NSRR-PyTorch
# Dev arch

### What

* Used a pytorch template to create basic project architecture.
* Added Dataset and DataLoader classes responsible for loading View, Depth and Motion images.
* Added FeatureExtraction module and loss function described in the paper.
* Implemented backward warping functionality.
* Added a script to test each functionality.

### How to test

1. Just run `unit_test.py` as described in README. Just make sure you're in a pytorch environment.
